release.go

```diff
@@ -227,6 +234,26 @@ func buildReleaseCommits(commits []Commit, cfg ReleaseConfig) []ReleaseCommit {
 		}
 	}
 
+	gen, limit := cfg.ReleaseNoteGenerator, 1000
+	prs := make(map[int]*github.PullRequest, limit)
+	if gen.UsePullRequestMetadata {
+		opts := &ListPullRequestOptions{
+			State:     PullRequestStateClosed,
+			Sort:      PullRequestSortUpdated,
+			Direction: PullRequestDirectionDesc,
+			Limit:     limit,
+		}
+		v, err := ghClient.listPullRequests(ctx, event.Owner, event.Repo, opts)
+		if err != nil {
+			log.Fatalf("Failed to list pull requests: %v\n", err)
```

Comment:
nit: Avoid using log.Fatal, instead we can return the error immediately.
(Only one place we can use that function is inside the main function.)